.taprc

```diff
@@ -1,4 +1,3 @@
-esm: false
```

Comment:
This PR is going to require _way_ more than just this one deletion. The `beforeEach` and `afterEach` methods no longer support callbacks. Please see https://github.com/tapjs/node-tap/pull/728#issue-601157222 for all breaking changes.